tslint.json

```diff
@@ -9,7 +9,8 @@
         "no-console": false,
         "no-shadowed-variable": false,
         "ordered-imports": false,
-        "no-string-literal": false
+        "no-string-literal": false,
+        "no-bitwise": false
```

Comment:
When we change the bitwise enum stuff above we may be able to remove this... i don't know if this flag limits all bitwise operators or just a subset.